src/source/PeerConnection/PeerConnection.h

```diff
@@ -99,6 +99,8 @@ VOID onSctpSessionDataChannelOpen(UINT64, UINT32, PBYTE, UINT32);
 
 STATUS sendPacketToRtpReceiver(PKvsPeerConnection, PBYTE, UINT32);
 
+STATUS generateJSONSafeString(PCHAR, UINT32);
```

Comment:
Is it a good pattern to expose functions that aren't used outside of a compilation file?

Reply:
Not the best idea really. Even for the tests.. Let me fix this asap